src/components/DetailsSearch.vue

```diff
@@ -90,34 +104,54 @@ export default {
           return;
         }
 
+        const item = {
+          object: codeObject,
+          childrenCount: codeObject.events ? codeObject.events.length : 0,
+        };
+
         switch (codeObject.type) {
           case 'package':
-            if (codeObject.children.length > 1) {
-              items.code.data.push(codeObject);
+            if (codeObject.childLeafs().length > 1) {
+              items[codeObject.type].data.push(item);
             }
             break;
-          case 'function':
-            items.code.data.push(codeObject);
-            break;
           case 'class':
             if (codeObject.functions.length) {
-              items.code.data.push(codeObject);
+              items[codeObject.type].data.push(item);
             }
             break;
+          case 'function':
           case 'route':
-            items.http.data.push(codeObject);
-            break;
           case 'query':
-            codeObject.events.forEach((e) => items.sql.data.push(e));
+            items[codeObject.type].data.push(item);
             break;
           default:
             break;
         }
       });
 
-      items.labels.data = Object.keys(this.appMap.labels).filter((l) =>
-        this.passesFilter(l)
-      );
+      Object.entries(this.appMap.labels).forEach(([label, labelData]) => {
+        if (this.passesFilter(label)) {
+          items.labels.data.push({
+            object: label,
+            childrenCount: labelData.function.length + labelData.event.length,
+          });
+        }
+      });
+
+      Object.values(items).forEach((item) => {
+        item.data = item.data.sort((a, b) => {
+          const aStr =
+            a.object instanceof CodeObject
+              ? a.object.prettyName
+              : String(a.object);
+          const bStr =
+            b.object instanceof CodeObject
+              ? b.object.prettyName
+              : String(b.object);
+          return aStr.localeCompare(bStr);
```

Comment:
@dustinbyrne This comparision doesn't work well with `Query` code objects, because when we call `object.prettyName` it returns `SQL Select`/`SQL Insert`/`SQL Update`, etc.
Are we okay with that?

Reply:
I'm OK with this, I think we need another method to sort queries anyway.